### PR TITLE
Fix broken link in Overview.md

### PR DESCRIPTION
**Pull request by jiyeonseo**
**Summary**

fixed the broken link on the documentation

**Test Plan**
.

### CODE REVIEW

**Comment by claudiopro** [comment]
Thanks for spotting this, @jiyeonseo! Do you mind updating the link href to the actual permalink?

https://conf2016.reactjs.org/schedule.html#rich-text-editing-with-react

Thanks!

File: docs/Overview.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Draft.js is a framework for building rich text editors in React, powered by an i
 
 Draft.js makes it easy to build any type of rich text input, whether you're just looking to support a few inline text styles or building a complex text editor for composing long-form articles.
 
-Draft.js was introduced at [React.js Conf](http://conf.reactjs.com/) in February 2016.
+Draft.js was introduced at [React.js Conf](https://conf.reactjs.org/) in February 2016.
```
Comment (claudiopro) [inline docs/Overview.md]: Please change the URL to https://conf2016.reactjs.org/schedule.html#rich-text-editing-with-react

Comment (jiyeonseo) [reply docs/Overview.md]: @claudiopro Changed :)

**Comment by claudiopro** [comment]
Super, thank you @jiyeonseo ✨

**Pull request closed without merging**